getpaid/forms.py

```diff
@@ -3,7 +3,11 @@
 from django.forms import forms
 from django.forms.fields import ChoiceField, CharField
 from django.forms.models import ModelChoiceField
-from django.forms.widgets import HiddenInput, RadioSelect, RadioFieldRenderer, RadioChoiceInput
+from django.forms.widgets import HiddenInput, RadioSelect, RadioFieldRenderer
+try:
+    from django.forms.widgets import RadioInput
+except ImportError:
+    from django.forms.widgets import RadioChoiceInput as RadioInput
```

Comment:
@piotrgiedziun since 1.8 and 1.9 have RadioChoiceInput could you do it the other way round please?